crates/swc_ecma_transforms_module/src/system_js.rs

```diff
@@ -562,6 +562,12 @@ impl Fold for SystemJs {
                 let assign_expr = AssignExpr {
                     right: match *assign.right {
                         Expr::Ident(ident) => Box::new(self.fold_module_name_ident(ident)),
+                        Expr::Assign(_) if assign.op == AssignOp::Assign => {
```

Comment:
Please use a pattern instead of an if guard. e.g. `Expr::Assign(AssignExpr{ op: ... })`

Reply:
Got it! I have modified my code